public/bulls-eye.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this file?  Please remove if it's not used in the current PR (it looks like it was for the bullets you were working on?).